src/Compilers/CSharp/Test/Symbol/Symbols/GenericConstraintTests.cs

```diff
@@ -5810,7 +5809,59 @@ class @c
 ";
             // NOTE: we don't report that object* and void* are invalid type arguments, since validation
             // is performed on A.I, not on F<object*>.I or G<void*>.I.
-            CreateCompilation(source).VerifyDiagnostics();
+            CreateCompilation(source, parseOptions: TestOptions.Regular11).VerifyDiagnostics();
+        }
+
+        [Fact, WorkItem(11243, "DevDiv_Projects/Roslyn")]
+        public void ConstraintGenericForPoint_CSharp12_A()
+        {
+            var source = @"
```

Comment:
For test variations where the source is identical, it is preferable to use a single test and just call CreateCompilation twice with different settings.